### PR TITLE
Fireeyehx search

### PR DESCRIPTION
**Pull request by shaniacht1**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/11590

## Description
added the search command that used to not work on older versions of FireEYE.
Added get-host-sets command

## Screenshots
Paste here any images that will help the reviewer

## Related PRs
List related PRs against other branches:

branch | PR
------ | ------

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

### CODE REVIEW

**Comment by anara123** [comment]
@noaco @shaniacht1 please review and merge

File: scraping.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    }
+
+    try:
+        jsonRes = response.json()
+        if (type(jsonRes) is not list) or (len(jsonRes) <= 0) or (jsonRes[0].get('right', None) is None):
+            # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+            raise "error!!!! " + response.text
+    except Exception as err:
+        # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+        print "some error " + str(err)
```
Comment (noaco) [inline scraping.py]: you are printing the errors and not using the commented error entries on purpose?

File: Integrations/integration-FireEyeHX.yml [hunk 2]
```diff
@@ -1752,7 +1820,7 @@ script:
             'ReadableContentsFormat': formats['markdown'],
             'HumanReadable': '## Search Results\n' + '\n'.join(md_entries),
             'EntryContext': {
-                "FireEyeHX.Search(obj._id==val._id)": results
+                "FireEyeHX.Search": search_results_to_context(results, search_id)
```
Comment (noaco) [inline Integrations/integration-FireEyeHX.yml]: can we pull the results again a specific search? if so maybe add (obj._id==val._id)

File: scraping.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+requests.packages.urllib3.disable_warnings()
+
+def validateSuccessfulResponse(resp):
+    if resp.status_code != 200:
+        # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+        print "error!!!! " + resp.text
+        exit(0)
```
Comment (noaco) [inline scraping.py]: sys.exit(0)

File: scraping.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+    data = json.dumps(data)
+    response = session.post(restAddress+'/resultByReport?start=0&perPage='+max_events+'&allData='+extendedData, headers=headers, data=data, verify=False)
+
+    try:
+        res = response.json()
+        eventKeys = res["headerData"]["columnNames"]
+    except Exception as err:
+        # demisto.results(errorEntry("Got response status " + str(resp.status_code) + " when fetching event query"))
+        print "some error " + str(err) + res.text
+        exit(0)
```
Comment (noaco) [inline scraping.py]: sys.exit(0)

**Comment by anara123** [comment]
@shaniacht1 please fix and merge

**Pull request merged**